examples/acm-terraform-blog-part1/terraform/gke.tf

```diff
@@ -0,0 +1,32 @@
+module "enabled_google_apis" {
+  source  = "terraform-google-modules/project-factory/google//modules/project_services"
+  version = "~> 10.0"
+
+  project_id                  = var.project
+  disable_services_on_destroy = false
+
+  activate_apis = [
+    "compute.googleapis.com",
+    "container.googleapis.com",
+    "gkehub.googleapis.com",
+    "anthosconfigmanagement.googleapis.com"
+  ]
+}
+
+resource "random_id" "rand" {
+  byte_length = 4
+}
+
+module "gke" {
+  source                = "terraform-google-modules/kubernetes-engine/google"
+  project_id            = var.project
+  name                  = "sfl-acm-${random_id.rand.hex}"
```

Comment:
nit: cluster is project scoped do we need a random id?

Reply:
Remove random id part, not required for this example.